DOCSP-35892: Quick Start docs (#2721)
jenssegers/laravel-mongodb: docs/quick-start/view-data.txt

.. laravel-quick-start-view-data:

========================
View Sample MongoDB Data
========================

.. facet::
   :name: genre
   :values: tutorial

.. meta::
   :keywords: test connection, runnable, code example

.. procedure::
   :style: connected

   .. step:: Create a Sample Model and Controller

      Create a model called ``Movie`` to represent data from the sample

>       ``movies`` collection in your MongoDB database. You can create the
>       model with a corresponding resource controller by running the
>       following command:

Reviewer: S: since this is a tutorial, I think it makes sense to replace "you can" with something more action oriented

```suggestion
      ``movies`` collection in your MongoDB database. Run the following command
      to create the model with a corresponding resource controller:
```

Author: Thanks for raising this. There's some nuance in that this is one way of accomplishing the task mentioned in the previous sentence. I'll update the wording to avoid offering the choice.
